### PR TITLE
Python-menus-toolbars-sample-app

### PR DESCRIPTION
**Pull request by lpozo**
**Where to put new files:**

- New files should go into a top-level subfolder, named after the article slug. For example: `my-awesome-article`

**How to merge your changes:** 

1. [Make sure the CI code style tests all pass (+ run the automatic code formatter if necessary).](https://github.com/realpython/materials/blob/master/README.md)
2. Find an RP Team member on Slack and ask them to review & approve your PR.
3. Once the PR has one positive ("approved") review, GitHub lets you merge the PR.
4. 🎉

### CODE REVIEW

File: python-menus-toolbars/MIT.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2014 Prahlad Yeri
```
Comment (bzaczynski) [inline python-menus-toolbars/MIT.txt:3]: @lpozo Shouldn't this be your name instead? :slightly_smiling_face:

File: python-menus-toolbars/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Sample PyQt Application
+
+A sample GUI application that shows how to create and use menus, toolbars, and status bars using Python and PyQt.
+
```
Comment (bzaczynski) [inline python-menus-toolbars/README.md:4]: @lpozo I'd include a short instruction for the impatient reader showing how to run this app:

```
$ pip install pyqt5
$ python sample-app.py
```

File: python-menus-toolbars/sample-app.py [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+        super().__init__(parent)
+        self.setWindowTitle("Python Menus & Toolbars")
+        self.resize(400, 200)
+        self.centralWidget = QLabel("Hello, World")
+        self.centralWidget.setAlignment(Qt.AlignHCenter | Qt.AlignVCenter)
+        self.setCentralWidget(self.centralWidget)
+        self._createActions()
+        self._createMenuBar()
+        self._createToolBars()
+        # self._createContextMenu()
```
Comment (bzaczynski) [inline python-menus-toolbars/sample-app.py:44]: @lpozo That line looks unnecessary until finding the related block of code later, which is also commented out. Can you add a note in a comment here as well to clear things up?

File: python-menus-toolbars/MIT.txt (hunk 1 above)
Comment (lpozo) [reply python-menus-toolbars/MIT.txt:3]: Ohh, sorry about that. Copy/Paste is a bad practice.

File: python-menus-toolbars/README.md (hunk 2 above)
Comment (lpozo) [reply python-menus-toolbars/README.md:4]: Done!

File: python-menus-toolbars/sample-app.py (hunk 3 above)
Comment (lpozo) [reply python-menus-toolbars/sample-app.py:49]: I added a comment about that. Let me know if it's okay with you. Thanks!

**Review by bzaczynski — APPROVED** [review]
@lpozo Thanks for a quick update. This looks good to me!

**Comment by bzaczynski** [comment]
lgtm

**Pull request merged**